docs/guides/composer.mdx

```diff
@@ -0,0 +1,731 @@
+---
+title: Explore gates and circuits with IBM Quantum Composer
+description: IBM Quantum Composer is a graphical programming tool. Drag and drop operations to build circuits and run them on quantum hardware.
+platform: cloud
+---
+
+{/* cspell:ignore gatename, ungroup, ungrouping */}
+# Explore gates and circuits with IBM Quantum Composer
+
+## What is it?
+
+[IBM Quantum&reg; Composer](https://quantum.cloud.ibm.com/composer) is a graphical quantum programming tool that lets you drag and drop operations to build quantum circuits and run them on quantum hardware.
+
+## What can it do?
+
+**Visualize qubit states**
+
+See how changes to your circuit affect the state of qubits, shown as an interactive q-sphere, or histograms showing measurement probabilities or statevector simulations.
+
+**Run on quantum hardware**
+
+Run your circuits on real quantum hardware to understand the effects of device noise.
+
+**Automatically generate code**
+
+Instead of writing code by hand, automatically generate OpenQASM or Python code that behaves the same way as the circuit you created with the Composer.
+
+## Tour of the interface
+
+IBM Quantum Composer has a customizable set of tools that allow you to build, visualize, and run quantum circuits on quantum processing units (QPUs). Use the More options menu on each window to access additional tools and actions.
```

Comment:
Should this be in quotes? I think it would make the English & translations clearer

```suggestion
IBM Quantum Composer has a customizable set of tools that allow you to build, visualize, and run quantum circuits on quantum processing units (QPUs). Use the "More options" menu on each window to access additional tools and actions.
```

Reply:
Ditto the rest of the file, like "View"

Reply:
To discuss with Becky when she gets back. I went through and put quotes around any menu/button label that uses a phrase rather than a single word ("More options", "Visualizations seed", etc.) and wasn't already emphasized otherwise in the sentence.